platform/components/insights/clusters/clusters-sheet.tsx

```diff
@@ -81,22 +83,86 @@ const RunClusters = ({
   const hobby = orgMetadata?.plan === "hobby";
 
   const [loading, setLoading] = React.useState(false);
-  const [scope, setScope] = React.useState("messages");
-  const [instruction, setInstruction] = React.useState("");
+
+  if (!project_id) {
+    return <></>;
+  }
```

Comment:
Put this at the bottom of the components, above the other return component
Otherwise it may cause errors due to varying number of hooks

Reply:
Done, thanks !